setup.cfg

```diff
@@ -49,3 +49,5 @@ exclude =
     __pycache__,
     .eggs,
     user_data,
+per-file-ignores =
+    freqtrade/freqai/RL/BaseReinforcementLearningModel.py:C901
```

Comment:
You can do excludes inline, in the file itself at the top of the file (or at the head of the function, if you'd prefer to exclude just one function).

doing it here is kinda odd - and will fail "at random" should we ever rename or move the file.

Reply:
Oh, if we can do it per function then that solves it. I will ask chatGPT to google that for me and then I will implement it.

Reply:
>  I will ask chatGPT to google that for me and then I will implement it. 

chatGPT can't really google
but if you search through the codebase for C901 you'll see that we have 2 instances of that already

https://github.com/freqtrade/freqtrade/blob/d47d8c135b0f3fcac430e1ad225c55d7465583a2/freqtrade/rpc/api_server/api_backtest.py#L29-L31

Reply:
>chatGPT can't really google

oh you thought that was serious hah.

Reply:
well - no 😆